packages/sk-marked/src/__tests__/utils.spec.js

```diff
@@ -0,0 +1,61 @@
+import { format } from '../utils';
+
+describe('format', () => {
```

Comment:
Last time I tried a test() inside describe() it didn't work as expected. Could be wrong now, but you don't really even need the describe here. Long-story short, I've been moving everything to just use test() and separate things into files as much as possible, mostly matching the src structure.